UnityProject/Assets/Scripts/Machines/Jukebox.cs

```diff
@@ -34,6 +34,12 @@ public class Jukebox : NetworkBehaviour, IAPCPowered
 	[SerializeField]
 	private SpriteDataSO SpriteDamaged;
 
+	[SerializeField]
+	private float MinSoundDistance;
+
+	[SerializeField]
+	private float MaxSoundDistance;
```

Comment:
Could you assign any non-public `[SerializeField]`s to some value, like `default` or `null`. Just helps reduce compiler warnings. Thanks.

Reply:
Done